docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+    {
+        /* After setting both description and ICE candidates from the remote peer, it'll be changed from WEBRTC_ICE_CONNECTION_STATE_CHECKING to WEBRTC_ICE_CONNECTION_STATE_COMPLETED. */
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        ...
+        ret = webrtc_set_peer_connection_state_change_cb(webrtc, _peer_connection_state_change_cb, user_data);
+        ret = webrtc_set_signaling_state_change_cb(webrtc, _signaling_state_change_cb, user_data);
+        ret = webrtc_set_ice_gathering_state_change_cb(webrtc, _ice_gathering_state_change_cb, user_data);
+        ret = webrtc_set_ice_connection_state_change_cb(webrtc, _ice_connection_state_change_cb, user_data);
+        ...
+        ret = webrtc_start(webrtc);
+        /* Do negotiation */
+    }
+    ```
+
+<a name="media_render"></a>
+## Rendering audio/video data
+
+You can decide how to handle audio/video streaming data received from a remote peer by using functions provided in this API set. You can also render sending audio/video data on the local target device.
+
+1. To get notified of creation of audio or video track from a remote peer, use `webrtc_set_track_added_cb()` function:
+
+    ```c
+    void _track_added_cb(webrtc_h webrtc, webrtc_media_type_e type, unsigned int track_id, void *user_data)
+    {
+        int ret;
+        some_app_data_s *data = (some_app_data_s *)user_data;
+
+        if (type == WEBRTC_MEDIA_TYPE_AUDIO) {
+            ret = sound_manager_create_stream_information(SOUND_STREAM_TYPE_MEDIA, NULL, NULL, &data->stream_info);
+            ret = webrtc_set_sound_stream_info(webrtc, track_id, data->stream_info);
+
+        } else if (type == WEBRTC_MEDIA_TYPE_VIDEO) {
+            /* To render video data to video overlay, use window id */
+            ret = webrtc_set_display(webrtc, id, WEBRTC_DISPLAY_TYPE_OVERLAY, data->win_id);
+            ... or ...
+            /* To render video data to EVAS image object */
+            ret = webrtc_set_display(webrtc, id, WEBRTC_DISPLAY_TYPE_EVAS, data->evas_image_object);
+        }
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        ...
+        ret = webrtc_set_track_added_cb(webrtc, _track_added_cb, data);
+        ...
+        ret = webrtc_start(webrtc);
+        ...
+        /* After finishing negotiation, _track_added_cb() could be called if receiving audio/video data from the remote peer exists */
+    }
+    ```
+    > [!IMPORTANT]
+    >
+    > `webrtc_set_sound_stream_info()` or `webrtc_set_display()` must be called inside of the callback set by `webrtc_set_track_added_cb()` if you want to output the audio or video track from the remote peer to the local target device's audio device or video display.
+
+2. To get media packet handle which packs the audio or video data from a remote peer, use `webrtc_set_encoded_audio_frame_cb()` or `webrtc_set_encoded_video_frame_cb()` function:
```

Comment:
Please avoid using the word "function" when function is already declared.
```suggestion
2. To get media packet handle which packs the audio or video data from a remote peer, use `webrtc_set_encoded_audio_frame_cb()` or `webrtc_set_encoded_video_frame_cb()`:
```

Reply:
Okay, done.